tests/repl/test_spread.rs

```diff
@@ -30,7 +30,7 @@ fn not_spread() -> TestResult {
     run_test(r#"def ... [x] { $x }; ... ..."#, "...").unwrap();
     run_test(
         r#"let a = 4; [... $a ... [1] ... (5) ...bare ...] | to nuon"#,
-        "[..., 4, ..., [1], ..., 5, ...bare, ...]",
+        r#"["...", 4, "...", [1], "...", 5, "...bare", "..."]"#,
```

Comment:
That `run_test` kind of escapes the meaning of that old test and is more one for the `nuon` sanity checks right?

Reply:
I think so, yes.